### PR TITLE
added firebase imports to docs

### PR DESCRIPTION
**Pull request by mason**
While going through the tutorial I was confused where `firebase` came from so I thought it would be helpful to add it to the examples. some of the other docs have imports so I thought these should also have them.

### CODE REVIEW

File: packages/documentation/docs/api/vuefire.md [hunk 1]
```diff
@@ -5,11 +5,15 @@ For all code samples, we will consider a `db` variable is imported as follows:
 <FirebaseExample id="db creation">
 
 ```js
+import firebase from 'firebase'
```
Comment (posva) [inline packages/documentation/docs/api/vuefire.md]: It should be a more specific import of `firebase/app`

File: packages/documentation/docs/api/vuefire.md [hunk 2]
```diff
@@ -5,11 +5,15 @@ For all code samples, we will consider a `db` variable is imported as follows:
...
+
 // Get a RTDB instance
 const db = firebase.initializeApp({ databaseURL: 'MY PROJECT URL' }).database()
 ```
 
 ```js
+import firebase from 'firebase'
```
Comment (posva) [inline packages/documentation/docs/api/vuefire.md]: here, it also needs firestore imports

File: packages/documentation/docs/api/vuefire.md (hunk 1 above)
Comment (mason) [reply packages/documentation/docs/api/vuefire.md]: Hi, I couldn't get my app working with simply `import firebase from "firebase/app"`
I had to also include `import "firebase/firestore"`. Would like to learn more about how this works as I am pretty new to JS.

File: packages/documentation/docs/api/vuefire.md (hunk 2 above)
Comment (mason) [reply packages/documentation/docs/api/vuefire.md]: turns out I missed a lot

File: packages/documentation/docs/api/vuefire.md (hunk 1 above)
Comment (posva) [reply packages/documentation/docs/api/vuefire.md]: yeah, you need to import multiple things, you should be able to find detailed explanations on Firebase docs

Comment (mason) [reply packages/documentation/docs/api/vuefire.md]: Cool. I looked at the source code and importing the subpackages essentially calls App.registerService which hooks into vue ecosystem. https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/index.ts#L29

anyways I think I made all the changes. I put import firestore and import database where required in the docs.

**Comment by mason** [comment]
hey let me know if I need to do anything else with this PR! thanks.

**Comment by mason** [comment]
> Sorry for the delay, could you use single quotes everywhere please?

thanks done.

**Pull request merged**